components/tag/style/index.ts

```diff
@@ -67,10 +65,10 @@ const genBaseStyle = (token: TagToken): CSSInterpolation => {
       height: 'auto',
       marginInlineEnd: token.marginXS,
       paddingInline,
-      fontSize: token.tagFontSize,
-      lineHeight: `${token.tagLineHeight}px`,
+      fontSize: token.fontSizeSM,
+      lineHeight: token.lineHeight,
```

Comment:
```suggestion
      lineHeight: token.lineHeightSM,
```
主要是这里不对哈，建议还是把原来的自定义 token 加回去，看起来比较清楚些

Reply:
另外自定义 token 的 tag 前缀要留着

Reply:
done